crates/symbolicator-service/src/caching/cache_key.rs

```diff
@@ -15,7 +15,7 @@ use crate::types::Scope;
 #[derive(Debug, Clone, Eq)]
 pub struct CacheKey {
     scope: Scope,
-    metadata: Arc<str>,
+    pub(crate) metadata: Arc<str>,
```

Comment:
maybe we should just expose this as a (public) method `fn as_str(&self) -> &str` and can also rename the field, so it doesn't collide with our actual metadata (like we discussed a while ago)?

Reply:
The method exists already; I made this `pub(crate)` so I could clone the `Arc`. Have now decided to just use a string.